email/1.3/modules/ROOT/edits/email-gmail.adoc

```diff
@@ -0,0 +1,160 @@
+= Connect to Gmail with Email Connector Examples - Mule 4
+:page-aliases: connectors::email/email-gmail.adoc
+
+To connect Anypoint Connector for Email (Email Connector) to your Gmail account, enable *Less Secure Apps* in your account and configure a global element for the server connection IMAPS, POP3S, or SMTPS you want to use.
+
+== Enable Less Secure Apps in your Gmail Account
+
+Google might block third-party mail clients if you do not enable the *Less Secure Apps* setting for your Gmail account, as Google blocks any apps that are not using security protocols that Google deems mandatory unless you allow less secure apps to access your Gmail account.
```

Comment:
GLOBAL "as" > "because" if that is what you mean (translation):
```suggestion
Google might block third-party mail clients if you do not enable the *Less Secure Apps* setting for your Gmail account, because Google blocks any apps that are not using security protocols that Google deems mandatory unless you allow less secure apps to access your Gmail account.
```
Here's another wording to consider:
```suggestion
To prevent Google from blocking third-party mail clients that are not using security protocols that Google deems mandatory, enable the *Less Secure Apps* setting for your Gmail account.
```